package.json

```diff
@@ -9,6 +9,7 @@
         "build": "webpack"
     },
     "dependencies": {
+        "@material-ui/core": "^4.11.0",
```

Comment:
Material-UI is too big package.
So, how about using the package below, if you just intend to use progress.
[https://github.com/davidhu2000/react-spinners](url)